int32 trig
int32 echo
---
sr04us/PingResponse[] readings
